src/use/ws.ts

```diff
@@ -9,10 +9,6 @@ import {
 } from '../common';
 import { limitCloseReason } from '../utils';
 
-// for nicer documentation
-type WebSocket = typeof ws.prototype;
-type WebSocketServer = ws.Server;
-
```

Comment:
These should be put back because they're used for documentation.

```suggestion
// for nicer documentation
export type { WebSocket, WebSocketServer };
```

Reply:
Done :)